frontend/src/components/pages/events/EventDetails.tsx

```diff
@@ -346,6 +348,31 @@ const EventDetails: React.FC<Props> = ({ eventId }) => {
                     </Typography>
                   </>
                 )}
+                <Button
+                  variant="text"
+                  href={window.URL.createObjectURL(
+                    new File(
+                      [
+                        `BEGIN:VCALENDAR\nCALSCALE:GREGORIAN\nMETHOD:PUBLISH\nPRODID:-//Test Cal//EN\nVERSION:2.0\nBEGIN:VEVENT\nUID:\nDTSTART;VALUE=DATE:${dayjs(
+                          eventData.event.startTime
+                        )
+                          .locale(nb)
+                          .format("YYYYMMDDTHHmmss")}\nDTEND;VALUE=DATE:${dayjs(endTime)
+                          .locale(nb)
+                          .format("YYYYMMDDTHHmmss")}\nSUMMARY:${
```

Comment:
When you do the formatting manually, I believe you do not need to call `.locale(nb)` as it is primarily used for formatting and translation purposes. Otherwise, you can set the default locale by calling `dayjs.locale(nb)` after importing. See [`frontend/src/components/pages/events/CountdownButton.tsx`](https://github.com/hovedstyret/indok-web/blob/main/frontend/src/components/pages/events/CountdownButton.tsx) for an example of this.